src/DetailsView/reports/components/automated-check-issue-details-list.tsx

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+import { NamedSFC } from '../../../common/react/named-sfc';
+import { DecoratedAxeNodeResult } from '../../../injected/scanner-utils';
+import { AutomatedChecksIssueDetailsInstances } from './automated-check-issue-details-instances';
+import { BaseCardLayer } from './base-card-layer';
+
+export interface AutomatedChecksIssueDetailsListProps {
+    nodeResults: DecoratedAxeNodeResult[];
+}
+
+export const AutomatedChecksIssueDetailsList = NamedSFC<AutomatedChecksIssueDetailsListProps>('AutomatedChecksIssueDetailsList', props => {
+    return (
+        <BaseCardLayer>
+            <div className="report-instances">
+                <table className="report-instance-table">
+                    <tbody>
+                        <AutomatedChecksIssueDetailsInstances nodeResults={props.nodeResults} />
```

Comment:
why not `<AutomatedChecksIssueDetailsInstances {...props} />`??? This let you follow the deps pattern, if you add something to the child properties, this will not need to change.

Reply:
I was being particular about what props i am passing, this doesn't really make a difference now as my expectation is that the lower level component wont need any deps and only data from the props.

Reply:
what if the data change?

Reply:
what if you add another property? or remove one? or rename one? The snippet I commented before is not only shorter than the actual code, it's also more resilient to the changes I just mentioned.